src/ThemedStyleSheet.js

```diff
@@ -31,22 +33,14 @@ function get() {
 }
 
 function resolve(...styles) {
-  if (
-    process.env.NODE_ENV !== 'production'
```

Comment:
The tradeoff here is that now bundlers can’t delete the code, they have to always import a function and call it.

Reply:
Agreed, I think we should leave this inline so that it can be completely minified out in production builds.

Reply:
Thank you for your feedback. I wanted to keep it just a little bit cleaner. But now I understand that it was not the best approach

Reply:
Thanks for the PR!